tools/linter_lib/pep8.py

```diff
@@ -17,9 +17,9 @@ def run_pycodestyle(files, ignored_rules):
         color = next(colors)
         pep8 = subprocess.Popen(
             ['pycodestyle'] + files + ['--ignore={rules}'.format(rules=','.join(ignored_rules))],
-            stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True)
```

Comment:
Can you explain what was the buffer error and how removing `universal_newlines=True` solves the issue?